components/navigation/communityItems.js

```diff
@@ -2,17 +2,20 @@ import IconTools from '../icons/Tools'
 import IconGithubOrganization from '../icons/GithubOrganization'
 import IconSlack from '../icons/Slack'
 import IconContributing from '../icons/Contributing'
+import IconDashboard from '../icons/Dashboard'
 import IconTSC from '../icons/TSC'
 import IconMeetings from '../icons/Meetings'
 import IconNewsroom from '../icons/Newsroom'
 
 
 export default [
+
   { icon: IconTools, title: 'Tools & Services', href: '/docs/community/tooling', description: 'Explore the tools and services our awesome community has created.' },
   { icon: IconGithubOrganization, title: 'Github Organization', href: 'https://github.com/asyncapi', target: '_blank', description: 'Want to sneak in the code? Everything we do is open-sourced in our Github organization.' },
   { icon: IconSlack, title: 'Slack Workspace', href: 'https://asyncapi.com/slack-invite', target: '_blank', description: `Need help? Want to share something? Join our Slack workspace. We're nice people :)` },
   { icon: IconContributing, title: 'Contributing', href: 'https://github.com/asyncapi?type=source#-contribute-to-asyncapi', target: '_blank', description: `We are always welcoming and looking for contributions. If you are interested check out our contribution guide.` },
   { icon: IconTSC, title: 'Technical Steering Committee', href: '/community/tsc', description: 'Get to know what is a TSC member, how you can become one, and meet our current TSC members.' },
+  { icon: IconDashboard, title: 'Dashboard',href: '/community/dashboard',target: '_blank', description: `Just need a good first issue to start your contribution journey? or want to see what topics are hot in discussion?`},
```

Comment:
I don't think we need `target: '_blank'` as it is part of the same website.
interesting though is that is actually do not open in another tab, at least for me, in Brave